ckks/bootstrapping/utils.go

```diff
@@ -0,0 +1,7 @@
+package bootstrapping
+
+// pointyInt re-implements point.Int() of https://github.com/openlyinc/pointy because this
+// helper pacakge requires Go v1.18 whereas Lattigo is backward compatible with up to Go v1.14.
+func pointyInt(x int) *int {
```

Comment:
This is quite a standard method, not sure this requires credits ;)

Reply:
done